snapshot-labs/sx-monorepo
# fix(ui): fix missing discussion link when preview is unavailable

### Summary

When discussion link preview is unavailable, nothing is shown.
It should default to show the bare link

Closes: #86

### How to test

Before the fix:

1. Go to a proposal with an invalid discussion link (e.g. https://testnet.snapshotx.xyz/#/s-tn:wan-test.eth/proposal/0x04a142d72b0d9b004965be01c3eff9bdad7888ceccf7e053cde129ba92a4ab03)
2. It should not be able to show the discussion preview and link, and just the title like 

![Screenshot 2024-02-26 at 23 14 14](https://github.com/snapshot-labs/sx-monorepo/assets/495709/4a3b2f39-eee8-4c62-9d21-791a7ca017df)

After the fix:

1. Go to a proposal with an invalid discussion link (e.g. https://testnet.snapshotx.xyz/#/s-tn:wan-test.eth/proposal/0x04a142d72b0d9b004965be01c3eff9bdad7888ceccf7e053cde129ba92a4ab03)
2. It should show the bare discussion link, without the preview like:

![Screenshot 2024-02-26 at 23 11 45](https://github.com/snapshot-labs/sx-monorepo/assets/495709/7ababd1e-871f-4558-9fbe-468522c5b25b)

In proposal Editor, it should show the preview only when the link is valid, and show nothing otherwise
